fl4health/losses/deep_mmd_loss.py

```diff
@@ -240,7 +242,10 @@ def train_kernel(self, X: torch.Tensor, Y: torch.Tensor) -> None:
         self.sigma_phi_opt.requires_grad = True
         self.epsilon_opt.requires_grad = True
 
-        features = torch.cat([X, Y], 0)
+        indices = torch.randperm(Y.size(0))
```

Comment:
Maybe just add a small comment as to what we're doing here? That is, we're shuffling the examples to ensure they aren't always presented in the same order for training.